src/s11/handlers/s11_MB_resp_handler.c

```diff
@@ -29,7 +29,7 @@ extern struct GtpV2Stack* gtpStack_gp;
 /*End : globals and externs*/
 
 int
-s11_MB_resp_handler(MsgBuffer* message, GtpV2MessageHeader* hdr)
+s11_MB_resp_handler(MsgBuffer* message, GtpV2MessageHeader* hdr, uint32_t sgw_ip)
```

Comment:
this sgw_ip is not used at all...Not sure the purpose. Original code written by @badhrinathpa  was readable.